### ISSUE TITLE
[Website] In High Contrast Mode, the Firefox Private Relay website is not properly displayed

### ISSUE THREAD

**Issue by sbadau** [mozilla/fx-private-relay#192]
**Affected versions**
- Nightly 77.0a1
- Private Relay add-on version 1.2.0 - https://drive.google.com/file/d/1OqZCFEwHrbUTH0q1dQUJBI43qVBaM3Hw/view
                                                                                                                                                                                                                                             
**Affected platforms**
- Windows 10


**Steps to reproduce**
1. Enable the High Contrast mode 
2. Launch the latest Nightly 77.0a1
3. Go to "https://relay.firefox.com" and login to your account.
4. Observe the Firefox Private Relay website

**Expected behavior**
The Firefox Private Relay website should be properly displayed.

**Actual behavior**
The Firefox Private Relay website is not properly displayed in high contrast mode: the toggle button for forwarding emails is not visible and there is no delimitation between the aliases and the rest of the page (as it is in normal mode).
Please see the screenshot for more details:

![2020-04-23_18h22_24](https://user-images.githubusercontent.com/56026823/80119767-cda3b980-8592-11ea-87f0-5f1b56ef9fc9.png)

**Comment by sbadau** [comment]
This issue is reproducible also on the latest website (tested on Windows 10). Please see the screenshot

![highContrast](https://user-images.githubusercontent.com/56026823/84793652-0fcf0180-affe-11ea-8f6d-1bd17e82dc1f.png)
.

**Comment by groovecoder** [comment]
This should be fixed now.
